src/python.cc

```diff
@@ -224,12 +227,58 @@ class ModelState : public BackendModel {
   // Is decoupled API being used.
   bool IsDecoupled() { return decoupled_; }
 
+  // Create the model stub process.
+  TRITONSERVER_Error* SetupModelStubProcess();
+  // Start stub process
+  TRITONSERVER_Error* StartModelStubProcess();
+
+  // Auto-complete the model configuration
+  TRITONSERVER_Error* AutoCompleteConfig();
+  TRITONSERVER_Error* FixIO(
+      triton::common::TritonJson::Value& reference_ios,
+      triton::common::TritonJson::Value* mutable_ios);
+  TRITONSERVER_Error* CheckModelConfigMismatch();
+  TRITONSERVER_Error* ValidateModelConfig();
+
+  // Correct the string to the right json string format
+  void CorrectStringFormat(std::string* str);
+
+  common::TritonJson::Value& GetAutoCompleteConfig()
+  {
+    return auto_complete_config_;
+  }
+
  private:
   ModelState(TRITONBACKEND_Model* triton_model);
   BackendState* backend_state_;
   std::string python_execution_env_;
   bool force_cpu_only_input_tensors_;
   bool decoupled_;
+
+  // Stub process pid
+  pid_t stub_pid_;
```

Comment:
Can we refactor the stub functionality used in the ModelState and ModelInstanceState to another class named `StubLauncher` to avoid code duplication?